synfig-studio/src/gui/trees/layertree.cpp

```diff
@@ -152,33 +140,8 @@ LayerTree::LayerTree():
 	attach(*layer_amount_hscale, 1, 2, 1, 2, Gtk::EXPAND|Gtk::FILL, Gtk::SHRINK, 1, 1);
 	layer_amount_adjustment_->signal_value_changed().connect(sigc::mem_fun(*this, &studio::LayerTree::on_amount_value_changed));
 
-	Gtk::Image *icon;
 	Gtk::IconSize iconsize(Gtk::ICON_SIZE_SMALL_TOOLBAR);
 
-	SMALL_BUTTON(button_raise,"gtk-go-up","Raise");
```

Comment:
It seems this code has not been commented out, but I can’t figure out where these buttons were?

Reply:
They're not used.
The buttons are in the correspondent dock class

Reply:
Ok